docs/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "scripts": {
     "dev": "babel-node start.js",
-    "build:dev": "babel-node build.js && cd .. && yarn storybook:build && yarn examples:build",
+    "build:dev": "babel-node build.js && (cd .. && yarn storybook:build && yarn examples:build) && cp _redirects ./dist/",
```

Comment:
I was going to say 
> this should be on build rather than build:dev no?
but I just noticed Netlify actually ran build:dev. Isn't that a little strange?
probably not worth changing a this point though 😄

Reply:
Yeah I thought it was weird but that's how it is right now (not worth changing now).

Reply:
It's doing build:dev because the prod build was too long and it times out

Reply:
The `build:dev` already turned my laptop's fan like crazy haha